### PR TITLE
use barge instead of ganache

### PR DESCRIPTION
**Pull request by alexcos20**
For consistency, let's use barge for unit tests as well
You can say that it's over engineered, but if we need to change the mnemonic, or activate/deactivate EIPS, we are going to do that in one repo (barge)

### CODE REVIEW

**Comment by kremalicious** [comment]
please no. Please keep them separated so we do not wait for ages for every test. There is a reason I split them up and NOT using barge for the unit tests. If you need more than ganache for a test, it's an integration test,

**Comment by alexcos20** [comment]
instead of running ganache:latest with some extra param, we run barge with ganache and without aqua/provider/es, etc..

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -45,15 +45,34 @@ jobs:
           path: ~/.npm
           key: ${{ runner.os }}-test-unit-${{ env.cache-name }}-${{ hashFiles('**/package-lock.json') }}
           restore-keys: ${{ runner.os }}-test-unit-${{ env.cache-name }}-
-      - run: npm ci
-      - run: npm run build:metadata
 
-      - name: Run Ganache
+      # Env var expansion workaround
+      # https://docs.github.com/en/actions/reference/workflow-commands-for-github-actions#setting-an-environment-variable
+      - name: Set ADDRESS_FILE
```
Comment (kremalicious) [inline .github/workflows/ci.yml:51]: let's remove the `ADDRESS_FILE` job as it's not needed. Most likely the DcokerHub login won't be needed either

**Comment by kremalicious** [comment]
alright, so Ganache with Barge starts up faster than Ganache directly from DockerHub so let's go with that

**Review by kremalicious — APPROVED** [review]

**Pull request merged**